packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -139,15 +147,30 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       this.apiKey ? `&apiKey=${this.apiKey}` : ""
     ].join("");
 
-    const ohlcResponse = await this.networker.getJson(ohlcUrl);
-    if (!ohlcResponse || !ohlcResponse.result || !ohlcResponse.result[this.ohlcPeriod]) {
-      this.logger.error({
-        at: "CryptoWatchPriceFeed",
-        message: "Could not parse ohlc result🚨",
-        error: new Error(JSON.stringify(ohlcResponse))
-      });
-      return;
-    }
+    // 2. Send requests.
+    const [ohlcResponse, priceResponse] = await Promise.all([
+      this.networker.getJson(ohlcUrl),
+      this.networker.getJson(priceUrl)
+    ]);
+
+    // 3. Check responses.
+    assert(
```

Comment:
General question: This is the only place we use the `assert` syntax like this in the bots, outside of tests. I think it works well in this context but just wanted to hear what you think about consistency with this. Another option would be to generate a `new Error` if these conditions are not met.

Also, `assert`, if not met, generates an `Error` object, right?

Reply:
Yep, I was basically using it as a shortcut for:
```js
if (some_condition) {
  throw new Error("message");
}
```

And yeah, it generates an `AssertionError`, which I think is a type of Error object (it seems to have a message field and anything else you might expect from an error obj).

I'll change to the `if` syntax!

Reply:
Done. PTAL!